static/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Critter</title>
+    <link rel="stylesheet" href="/static/index.css">
+</head>
+<body>
+    <header>
+        <nav class="header-flex">
+            <img src="" alt="profile picture">
+            <h1>Home</h1>
+            <form action="post">
+                <label for="create-crit">Send a Crit!</label>
+                <input type="text" name="create-crit">
+            </form>
+        </nav>
+    </header>
+    
+    <main class="main-flex"> 
+        <div class="crit-timeline">
+            <h3>Some User @imSomeUser22</h3>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla numquam dolore rerum facere dicta repudiandae aperiam incidunt, blanditiis sed, voluptatem autem maxime animi amet architecto!</p>
+            <div class="btn-flex">
+                <button>Like</button>
```

Comment:
These buttons probably need classes so that we can attach javascript and styles to them in the future.